### PR TITLE
Logger

### PR DESCRIPTION
**Pull request by bandvov**

### CODE REVIEW

File: combined.log [hunk 1]
```diff
@@ -0,0 +1,107 @@
...   }\\n  }\\n}\\n\\nfragment FullType on __Type {\\n  kind\\n  name\\n  description\\n  fields(includeDeprecated: true) {\\n    name\\n    description\\n    args {\\n      ...InputValue\\n    }\\n    type {\\n      ...TypeRef\\n    }\\n    isDeprecated\\n    deprecationReason\\n  }\\n  inputFields {\\n    ...InputValue\\n  }\\n  interfaces {\\n    ...TypeRef\\n  }\\n  enumValues(includeDeprecated: true) {\\n    name\\n    description\\n    isDeprecated\\n    deprecationReason\\n  }\\n  possibleTypes {\\n    ...TypeRef\\n  }\\n}\\n\\nfragment InputValue on __InputValue {\\n  name\\n  description\\n  type {\\n    ...TypeRef\\n  }\\n  defaultValue\\n}\\n\\nfragment TypeRef on __Type {\\n  kind\\n  name\\n  ofType {\\n    kind\\n    name\\n    ofType {\\n      kind\\n      name\\n      ofType {\\n        kind\\n        name\\n        ofType {\\n          kind\\n          name\\n          ofType {\\n            kind\\n            name\\n            ofType {\\n              kind\\n              name\\n              ofType {\\n                kind\\n                name\\n              }\\n            }\\n          }\\n        }\\n      }\\n    }\\n  }\\n}\\n\"}","service":"user-service"}
```
Comment (Alan18081) [inline combined.log:1]: Please, add this file to the `.gitignore`, logs should not be pushed to the backend

File: error.log [hunk 2]
```diff
@@ -0,0 +1 @@
+{"level":"error","message":"Invalid authorization token","service":"user-service","timestamp":"2020-09-05T17:00:01.596Z"}
```
Comment (Alan18081) [inline error.log:1]: Please, add this file to the `.gitignore`, logs should not be pushed to the backend

File: logger.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+const logger = winston.createLogger({
+  level: 'info',
+  format: winston.format.combine(
+    winston.format.timestamp(),
+    winston.format.json()
+  ),
+  defaultMeta: { service: 'user-service' },
+  transports: [
+    //
+    // - Write all logs with level `error` and below to `error.log`
```
Comment (Alan18081) [inline logger.js:13]: Please, remove comments

File: server.js [hunk 4]
```diff
@@ -26,9 +27,19 @@ const server = new ApolloServer({
...
+      level: 'info',
+      message: `method: ${req.method}/baseUrl: ${req.baseUrl}/date:${
+        req.fresh
+      }/ request headers: ${JSON.stringify(req.headers)}/body: ${JSON.stringify(
+        req.body
+      )}`,
+    });
     if (token) {
       const user = verifyUser(token);
       if (!user) throw new AuthenticationError('Invalid authorization token');
```
Comment (Alan18081) [inline server.js:41]: Please, while you have time, adjust the error message to use `INVALID_AUTHORIZATION_TOKEN` in exception, tests if it is checked, and the logger message. I think it is a good idea to create a format function to remove the `_` and lowercase text from our standard error messages

**Review by maksym-strus — APPROVED** [review]

**Review by TarasKun — APPROVED** [review]

**Pull request merged**